pkg/graph/resolvers/mto_milestone.resolvers.go

```diff
@@ -40,11 +40,18 @@ func (r *mTOMilestoneResolver) SubCategory(ctx context.Context, obj *models.MTOM
 	panic(fmt.Errorf("not implemented: SubCategory - subCategory"))
 }
 
-// CreateMTOMilestone is the resolver for the createMTOMilestone field.
-func (r *mutationResolver) CreateMTOMilestone(ctx context.Context, modelPlanID uuid.UUID, name *string, commonMilestoneKey *models.MTOCommonMilestoneKey, mtoCategoryID *uuid.UUID) (*models.MTOMilestone, error) {
+// CreateMTOMilestone is the resolver for the createMTOMilestoneCustom field.
```

Comment:
```suggestion
// CreateMTOMilestoneCustom is the resolver for the createMTOMilestoneCustom field.
```

Reply:
That's actually kinda interesting that gqlgen didn't update these for us, since I'm pretty sure those comments are auto-generated. I'll update it either way!